src/queries/SearchResultQuery.ts

```diff
@@ -48,6 +52,22 @@ class SearchResultQuery extends BaseQuery<SearchResult> implements SearchResultQ
 
     this.filterQuery['keyword'] = { contains: this.keyword };
   }
+
+  private async buildUrlToFilterQueryParams() {
+    if (!this.urlToFilter) return;
```

Comment:
Lets add a line brak for readability

```suggestion
    if (!this.urlToFilter) return;
    
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/98/commits/2044255a548e5647eac03294d316b890738984be